source/Marquee.js

```diff
@@ -902,7 +902,7 @@
 			var node = this.$.marqueeText ? this.$.marqueeText.hasNode() : this.hasNode(),
 				rect;
 
-			if (node && this._marquee_distance == null) {
+			if (node && this._marquee_distance == null && this.getAbsoluteShowing() === true) {
```

Comment:
I think we can just check `this.getAbsoluteShowing()` instead of `this.getAbsoluteShowing() === true`.